lib/service.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
+
+/**
+ * Dependencies
+ */
+
+var format = require('util').format;
+var objectAssign = require('object-assign');
```

Comment:
It already is a ponyfill :). https://github.com/sindresorhus/object-assign/blob/master/index.js